test/queue.test.js

```diff
@@ -18,7 +18,7 @@ import { MockAwsClient } from './mocks/aws-client.js';
 
 const LARGE_MESSAGE = {
   // generates a 128KB string by joining the alphabet 5120 times
-  msg: [...Array(5120)].map(() => 'abcdefghiklmnopqrstuvwxyz').join(''),
+  msg: [...Array(5120)].map(() => 'abcdefghijklmnopqrstuvwxyz').join(''),
```

Comment:
for next time, i would suggest googling or asking chatgpt "spell out all the letters of the alphabet for me in a string" -- works everytime :)